src/main/java/marquez/service/models/Run.java

```diff
@@ -33,6 +33,10 @@
   @NonNull Run.State state;
   @Nullable Map<String, String> args;
 
+  public UUID getId() {
```

Comment:
_Minor:_ [Lombok](https://projectlombok.org) will generate getters/setters for us. So, we don't need to explicitly define one.